azure-quantum/azure/quantum/target/microsoft/__init__.py

```diff
@@ -0,0 +1,9 @@
+from .qio import (
```

Comment:
is this needed? my understanding is that this will allow a user to do `from azure.quantum.target.microsoft import ParallelTempering`, but I could be wrong.

Reply:
Yes that's right. I thought it'd be good to be consistent with the other providers, and be able to import `from azure.quantum.target.<provider> import <target>`, the fact that `microsoft` also has a subcategory `qio` and `fleet_management` may not be immediately obvious.